.travis.yml

```diff
@@ -1,71 +1,41 @@
-language: python
+language: c
```

Comment:
We can only run MacOS X if language is set to c or obj-c. In any case, we don't need to worry about the language since we use conda to set everything up, so we don't need python pre-installed.